test/cmd.js

```diff
@@ -343,6 +343,45 @@ describe('express(1)', function () {
     });
   });
 
+  describe('--ejs --view=pug', function () {
+    var ctx = setupTestEnvironment(this.fullTitle())
+
+    it('should create basic app with pug templates', function (done) {
+      run(ctx.dir, ['--view', 'pug'], function (err, stdout) {
```

Comment:
Though the description of the tests says "--ejs --view=pug", the actual command being executed is not that, but simply "--view pug"